.github/workflows/ci.yaml

```diff
@@ -21,112 +21,144 @@ jobs:
           - python_version: "3.10"
+            codename: jammy
             odoo_version: "16.0"
             odoo_org_repo: "odoo/odoo"
             image_name: py3.10-odoo16.0
+          - python_version: "3.10"
+            codename: jammy
+            odoo_version: "17.0"
+            odoo_org_repo: "odoo/odoo"
+            image_name: py3.10-odoo17.0
           # oca/ocb
           - python_version: "2.7"
+            codename: focal
             odoo_version: "8.0"
             odoo_org_repo: "oca/ocb"
             virtualenv_constraint: "<20.22"
             image_name: py2.7-ocb8.0
           - python_version: "2.7"
+            codename: focal
             odoo_version: "9.0"
             odoo_org_repo: "oca/ocb"
             virtualenv_constraint: "<20.22"
             image_name: py2.7-ocb9.0
           - python_version: "2.7"
+            codename: focal
             odoo_version: "10.0"
             odoo_org_repo: "oca/ocb"
             virtualenv_constraint: "<20.22"
             image_name: py2.7-ocb10.0
           - python_version: "3.5"
+            codename: focal
             odoo_version: "11.0"
             odoo_org_repo: "oca/ocb"
             virtualenv_constraint: "<20.22"
             image_name: py3.5-ocb11.0
           - python_version: "3.6"
+            codename: focal
             odoo_version: "12.0"
             odoo_org_repo: "oca/ocb"
             setuptools_constraint: "<58"  # vatnumber needs setuptools with 2to3 support
             virtualenv_constraint: "<20.22"
             image_name: py3.6-ocb12.0
           - python_version: "3.6"
+            codename: focal
             odoo_version: "13.0"
             odoo_org_repo: "oca/ocb"
             setuptools_constraint: "<58"  # vatnumber needs setuptools with 2to3 support
             virtualenv_constraint: "<20.22"
             image_name: py3.6-ocb13.0
           - python_version: "3.6"
+            codename: focal
             odoo_version: "14.0"
             odoo_org_repo: "oca/ocb"
             virtualenv_constraint: "<20.22"
             image_name: py3.6-ocb14.0
           - python_version: "3.8"
+            codename: focal
             odoo_version: "14.0"
             odoo_org_repo: "oca/ocb"
             image_name: py3.8-ocb14.0
           - python_version: "3.8"
+            codename: focal
             odoo_version: "15.0"
             odoo_org_repo: "oca/ocb"
             image_name: py3.8-ocb15.0
           - python_version: "3.9"
+            codename: focal
             odoo_version: "15.0"
             odoo_org_repo: "oca/ocb"
             image_name: py3.9-ocb15.0
           - python_version: "3.10"
+            codename: jammy
             odoo_version: "16.0"
             odoo_org_repo: "oca/ocb"
             image_name: py3.10-ocb16.0
+          - python_version: "3.10"
+            codename: jammy
+            odoo_version: "17.0"
+            odoo_org_repo: "oca/ocb"
+            image_name: py3.10-ocb17.0
```

Comment:
same remark as here : https://github.com/OCA/oca-addons-repo-template/pull/226#discussion_r1383368569 regarding python3.11. not sure it's relevant.

Reply:
3.10 is the minimum Python version we are going to support, se we test on that.
This aligns with Odoo's choice in https://github.com/odoo/odoo/pull/136904

Reply:
thanks for the answer.